src/main/java/seedu/address/ui/Tab.java

```diff
@@ -0,0 +1,9 @@
+package seedu.address.ui;
+
+/**
+ * Existing tabs.
+ */
+public enum Tab {
+    CLASSES,
```

Comment:
Perhaps you can add in your public static final int value here.

```
public enum Tab {
    CLASSES(0),
    ATTENDANCE(1)

    private final int value;
    Tab(int value) {
        this.value = value;
    }

    public int getValue() {
        this.value;
    }
}
```

Would it be a better design?

Reply:
Great suggestion! I will incorporate a getValue() method as u suggested above as well.

Reply:
Great suggestion! I will incorporate a getValue() method as u suggested above as well.